<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bridge Club</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/bridge-club.png</normaloff>:/images/bridge-club.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QListWidget" name="roundsList">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="4">
     <widget class="QTreeView" name="tablesList">
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>75</number>
      </attribute>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTreeWidget" name="statusList">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>59</number>
      </attribute>
      <column>
       <property name="text">
        <string>Table</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Rounds:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Device status:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_Tournament"/>
    <addaction name="actionExport_Tournament"/>
    <addaction name="actionLoad_local_tournament"/>
    <addaction name="separator"/>
    <addaction name="actionPlay_a_match"/>
    <addaction name="actionExport_selected_match"/>
    <addaction name="actionP_lay_selected_match"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEvent">
    <property name="title">
     <string>Event</string>
    </property>
    <addaction name="actionNew_Event"/>
    <addaction name="actionStop_Event"/>
    <addaction name="separator"/>
    <addaction name="actionCompute_Result"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEvent"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Event"/>
   <addaction name="actionCompute_Result"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Event">
   <property name="text">
    <string>&amp;New Event...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionStart_Server">
   <property name="text">
    <string>Start Server</string>
   </property>
  </action>
  <action name="actionCompute_Result">
   <property name="text">
    <string>Compute &amp;Result</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionImport_Tournament">
   <property name="text">
    <string>&amp;Import Tournament...</string>
   </property>
   <property name="toolTip">
    <string>Import Tournament</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExport_Tournament">
   <property name="text">
    <string>&amp;Export Tournament...</string>
   </property>
   <property name="toolTip">
    <string>Export Tournament</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPlay_a_match">
   <property name="text">
    <string>Import and &amp;Play a match...</string>
   </property>
   <property name="toolTip">
    <string>Import and Play a match</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionStop_Event">
   <property name="text">
    <string>S&amp;top Event</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionExport_selected_match">
   <property name="text">
    <string>Export selected &amp;match...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionLoad_local_tournament">
   <property name="text">
    <string>Load Local &amp;Tournament...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionP_lay_selected_match">
   <property name="text">
    <string>P&amp;lay selected match</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
